app/mailers/order_notifier.rb

```diff
@@ -0,0 +1,21 @@
+class OrderNotifier < ApplicationMailer
+
+  def customer(order)
+    @order = order
+
+    mail to: order.user.email, subject: 'Take-Away Order Receipt Confirmation' do |format|
+      format.html
+      format.text
+    end
+  end
+
+  def kitchen(order)
+    @order = order
+    
+    # CHANGE 'TO' ADDRESS TO PRODUCTION SETTING
+    mail to: 'patmbolger@gmail.com', subject: 'Order Received' do |format|
```

Comment:
kitchen email   and   the default from email    should be loaded from db
We should have a dashboard section in ActiveAdmin to make app wide config changes like these emails.